src/lib/the-graph.ts

```diff
@@ -1,3 +1,3 @@
 // IMPORTANT: we don't expose the key to the client
 export const fetchCrTheGraphEndpoint = `${process.env.THE_GRAPH_URL}/${process.env.THE_GRAPH_API_KEY}/${process.env.THE_GRAPH_ID}`
-export const fetchDaoTheGraphEndpoint = `${process.env.DAO_GRAPH_URL}/${process.env.DAO_GRAPH_API_KEY}/${process.env.DAO_GRAPH_ID}`
+export const fetchDaoTheGraphEndpoint = `${process.env.DAO_GRAPH_URL}/${process.env.THE_GRAPH_API_KEY}/${process.env.DAO_GRAPH_ID}`
```

Comment:
The DAO will have its own key, unless that @Freshenext know something that I dont know.

Reply:
@david-iov please bring it up with both DAO and TOK as we may end up using different keys (it depends on the expected number of requests)

Reply:
I guess I'd leave both variable for now and also add the `DAO_GRAPH_API_KEY` in AWS, and we can discuss this in another scope.